### PR TITLE
issue337 don't use dot files

### PR DESCRIPTION
**Pull request by xpusostomos**
A fix for issue 337:  Stop hiding the sidecar files for phrases and scripts #337

https://github.com/autokey/autokey/issues/337

json config files no longer hidden by "." in filename.

### CODE REVIEW

**Comment by BlueDrink9** [comment]
Thanks for the contribution
Off the bat, I'd suggest rebasing this off of the `develop` branch instead of master. Master is horribly out-of-date, and a feature change like this isn't suited to it. `develop` is the staging branch for `0.96.0`, anyway

File: lib/autokey/configmanager.py [hunk 1]
```diff
@@ -528,26 +544,33 @@ def get_serializable(self):
             }
         return d
 
+    def upgrade_config_files(self, data):
```
Comment (BlueDrink9) [inline lib/autokey/configmanager.py:249]: Good refactor, thank you

File: lib/autokey/common.py [hunk 2]
```diff
@@ -40,7 +40,7 @@
 
 APP_NAME = "autokey"
 CATALOG = ""
-VERSION = "0.95.10"
+VERSION = "0.96.0"
```
Comment (BlueDrink9) [inline lib/autokey/common.py:36]: This was what tipped me off to the fact that this was based off `master` instead of `develop`

**Review by BlueDrink9 — APPROVED** [review]
This looks good after my quick look-over. The main maintainer @luziferius might disagree, but I see nothing wrong with this outside of being based off a horribly-outdated branch.

**Comment by BlueDrink9** [comment]
Thanks for the contribution
Off the bat, I'd suggest rebasing this off of the `develop` branch instead of master. Master is horribly out-of-date, and a feature change like this isn't suited to it. `develop` is the staging branch for `0.96.0`, anyway

**Review by BlueDrink9 — CHANGES_REQUESTED** [review]
Accidentally approved instead of just commenting. Wait for the main maintainer to confirm, but I imagine you'll need to rebase this off `develop`

**Comment by xpusostomos** [comment]
Sorry, didn't realise I should be working off develop. How does this work, do I close this pull request and start a new one, or how does it work?

**Comment by xpusostomos** [comment]
Screwed up this one, creating a new one based on develop

**Comment by xpusostomos** [comment]
Screwed up this one, creating a new one based on develop

**Comment by BlueDrink9** [comment]
That's fine. I think what you could also do is force-push to your PR branch, after updating it locally. That would prevent duplicate PRs. Then just edit this PR on github so that it targets the `develop` branch instead of `master`

(I.e., locally do a `git branch issue337_orig` backup branch, then `git reset --hard upstream/develop`, then `git cherry-pick` the new commits you created and stored in `issue337_orig`, then `git push --force` which should update this PR.)

> Sorry, didn't realise I should be working off develop.

In future when contributing to open-source projects, check for a CONTRIBUTERS file in the root directory of the repo. ~That mentions to use `develop`~
EDIT: actually, that file only exists ***on*** `develop`, so that wouldn't have helped

**Pull request closed without merging**